middleware/topicsMiddleware.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection")
+
+exports.getAllTopicsModel = async () => {

```

Comment:
this is a model in the MVC view, not middleware if this could be renamed before you merge.